index.html

```diff
@@ -29,6 +29,7 @@
         <span id="current-version-span"></span>
     </button>
     <ul class="dropdown-menu" aria-labelledby="previousVersionDropdown">
+        <li><a class="dropdown-item" href="#" value="v4">Version 4</a></li>
```

Comment:
Why is there two version 3's? `v3.0 and v3`

Reply:
To support https://sop.oncokb.org/?version=v3.0 But I think I can make a slight change in script to avoid the dup. Let me work on that.